pipeline/vr/appendvrids.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+ARTIFACT_DIR="$1"
+INPUT_FILE="$2"
+OUTPUT_FILE="$3"
+IMAGE_NAME="${4:-brcachallenge/append-vr-ids:0.1}"
+SEQ_REPO_DIR="${5:-/usr/local/share/seqrepo}"
+
+# --- pre-step: build vr image (caching will make this fast if it was already built)
+docker build -t "${IMAGE_NAME}" .
+
+# ----------------------------------------------------
+# --- 1. spin up seqrepo-rest-service
+# ----------------------------------------------------
+docker run --rm -d \
+  --user=`id -u`:`id -g` \
+  --name seqrepo-service \
+  --network=host \
+  -v "${SEQ_REPO_DIR}":/usr/local/share/seqrepo \
+  biocommons/seqrepo-rest-service:latest
+
+# ...and wait for it to be available
+./wait-for localhost:5000 && echo " => ...seqrepo-rest-service is ready"
+
+# ----------------------------------------------------
+# --- 2. execute append-vr-ids
+# ----------------------------------------------------
+docker run --rm \
+  --name append-vr-ids \
+  --network=host \
+  -v "${ARTIFACT_DIR}":/artifacts \
+  -e UTA_DB_URL \
+  "${IMAGE_NAME}" \
+  -i "/artifacts/${INPUT_FILE}" \
+  -o "/artifacts/${OUTPUT_FILE}"
+
+# --- post-step: when vr's done, bring down its dependent services
+docker stop seqrepo-service
```

Comment:
This can lead to unexpected results if there are 2 pipelines running on the same machine. I would just leave the service running

Reply:
Agreed. For that matter, starting the service will fail if there's already an instance of it running, since it starts in host networking mode and would block the second instance from listening on port 5000. As you mentioned elsewhere, I suppose the service should be started just once via the `start-seqrepo-rest-service` target. If it fails to start due to the port already being bound (e.g., if a second pipeline run was initiated), then I suppose that failure should be ignored.

Reply:
ya.. im going to test this now. I'm planning to keep the docker run command to start the seqrepo-rest-service in appendvrids.sh so that it could hypothetically be run in isolation. I'm assuming that a failure to start the service because one is already running won't cause any issues.